actor/src/main/resources/reference.conf

```diff
@@ -1145,11 +1145,13 @@ pekko {
         # other platforms, will default to 1).
         ndots = default
 
-        # The policy used to generate dns transaction ids. Options are thread-local-random or secure-random.
-        # Defaults to thread-local-random similar to Netty, secure-random produces FIPS compliant random numbers but
-        # could block looking for entropy (these are short integers so are easy to bruit-force, use thread-local-random
-        # unless you really require FIPS compliant random numbers).
-        id-generator-policy = thread-local-random
+        # The policy used to generate dns transaction ids. Options are `thread-local-random`,
+        # `enhanced-double-hash-random` or `secure-random`. Defaults to `enhanced-double-hash-random` which uses an
+        # enhanced double hashing algorithm optimized for minimizing collisions with a FIPS compliant initial seed.
+        # `thread-local-random` is similar to Netty and secure-random produces FIPS compliant random numbers every time
+        # but could block looking for entropy (these are short integers so are easy to brute-force, use
```

Comment:
should be `secure-random` to match the others

Reply:
Fixed